test/client/getPaths.ts

```diff
@@ -1,107 +0,0 @@
-import addresses from "../fixtures/addresses.json";
```

Comment:
If you're deleting these files in this PR, wait until your sugar PR is merged before merging this one.